src/olympia/users/utils.py

```diff
@@ -87,7 +87,7 @@ def autocreate_username(candidate, tries=1):
 
 
 def system_addon_submission_allowed(user, parsed_addon_data):
-    guid = parsed_addon_data.get('guid') or ''
+    guid = parsed_addon_data.get('guid').lower() or ''
```

Comment:
Can you make that `parsed_addon_data.get('guid', '').lower()` ? Need to handle the case where somehow `guid` is not present in parsed data (mainly for tests)

Reply:
Yeah, I realised that after seeing the failed tests. 
Made a hotfix in a bit different way, is that one good or should I change to the method you mentioned?

Reply:
Perfect. Let's see if the tests pass now.